### ISSUE TITLE
Use '@' as selectTemplate case menu position error.

### ISSUE THREAD

**Issue by a365344743s** [zurb/tribute#467]
**How can we reproduce this bug?**

1.  config selectTemplate as below.
  `selectTemplate: function(item) {
            return '<span contenteditable="false">' + '@' + item.original.value + "</span>";
    }`
2.  Choose one @item
3. Tap BackSpace on keyboard
4. Tap LeftArrow  on keyboard
5. Tap RightArrow on keyboard

**What did you expect to happen?**
Do not show menu or fix menu position.

**What happened instead?**
Menu show at left-top.

**Screenshot:**
![image](https://user-images.githubusercontent.com/15245246/81678784-663ba400-9484-11ea-8a46-bcfeeed326b1.png)

I think the '@' mark in selectTemplate cause this problem.

**Comment by mrsweaters** [comment]
Oh interesting, so if you get rid of the `@` symbol, the problem goes away?

**Comment by a365344743s** [comment]
> Oh interesting, so if you get rid of the `@` symbol, the problem goes away?

Yes, if i remove `@` symbol in selectTemplate, everything is ok.

**Comment by JiangXiaoBai12** [comment]
How do you keep @ to solve this problem？

**Comment by a365344743s** [comment]
> How do you keep @ to solve this problem？

We just remove show @.

**Comment by JiangXiaoBai12** [comment]
How can we solve this problem if we keep the @ on display。I need to keep the @ on display

**Comment by JiangXiaoBai12** [comment]
And the trigger is also @

**Comment by JiangXiaoBai12** [comment]
How can we solve this problem if we keep the @  symbol in selectTemplate。what should I do? @mrsweaters @a365344743s

**Comment by Jony1943** [comment]
@lhb12345 我在React中是这么解决的
```jsx
this.refs.input.addEventListener("tribute-active-true", (e) => {
    setTimeout(() => {
        const tributePanels = document.getElementsByClassName('tribute-container');
        const lastPanel = tributePanels[tributePanels.length - 1];
        const { left, top } = lastPanel.style;
        if (left === '0px' || top === '0px') {
            lastPanel.style.display = 'none';
        }
    }, 10);
});
```
